src/test/ui/nll/issue-48697.stderr

```diff
@@ -0,0 +1,8 @@
+error: internal compiler error: unexpected region for local data ReFree(DefId(0/0:3 ~ issue_48697[317d]::foo[0]), BrAnon(0))
```

Comment:
@nikomatsakis : You say you are removing the ICE, but the expected stderr you have encoded here is an ICE?